civis/ml/_model.py

```diff
@@ -560,6 +561,9 @@ class ModelPipeline:
         Memory requested from Civis Platform for training jobs, in MiB
     disk_requested : float, optional
         Disk space requested on Civis Platform for training jobs, in GB
+    notifications : dict
+        See :func:`~civis.resources._resources.Scripts.post_custom` for
```

Comment:
@keithing I cribbed this from somewhere else -- does this look right to you? (I'm not as familiar with sphinx documentation as I should be)